client/asset/interface.go

```diff
@@ -118,7 +118,7 @@ type Wallet interface {
 	// during a swap. If the coin cannot be found for the coin ID, the
 	// ExchangeWallet should return CoinNotFoundError. This enables the client
 	// to properly handle network latency.
-	AuditContract(coinID, contract dex.Bytes) (AuditInfo, error)
+	AuditContract(coinID, contract, txData dex.Bytes) (AuditInfo, error)
```

Comment:
Maybe comment on how txData will be used.  I'm actually still wondering how exactly we will _robustly_ get confirmation that the rebroadcast we are planning "succeeded".  Presumably we'll decode the txn, validate it (against the provided contract script), and the we'll try to detect sendrawtransaction errors that just indicate the txn already exists, but I'm a little concerned there will be a variety of errors that look different but all stem from the tx already being known, and we may end up declaring audit failures improperly.  I guess we'll cross that bridge next.